clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -282,6 +282,7 @@ def err_drv_omp_host_target_not_supported : Error<
   "The target '%0' is not a supported OpenMP host target.">;
 def err_drv_expecting_fopenmp_with_fopenmp_targets : Error<
   "The option -fopenmp-targets must be used in conjunction with a -fopenmp option compatible with offloading, please use -fopenmp=libomp or -fopenmp=libiomp5.">;
+def err_drv_opt_incompatible_opt : Error<"The combination of '%0' and '%1' is not supported.">;
```

Comment:
Can you use `err_drv_option_conflict` instead?

Reply:
Yes, done.